### ISSUE TITLE
 Fix issues with Transformers 4.11.2

### ISSUE THREAD

**Issue by davidmezzetti** [neuml/txtai#118]
Transformers 4.11.2 has caused a couple breaking changes for txtai:

- Update labels pipeline to unwrap nested lists from zero-shot pipeline output
- Modify OnnxModel to return "logits" field as expected by text-classification and zero-shot classification pipelines

Also remove unneeded code in labels.py related to text-classification pipelines and use the HF pipeline directly.

**Comment by davidmezzetti** [comment]
- Update labels pipeline to unwrap nested lists from zero-shot pipeline output

This has been fixed in Transformers 4.11.3 and the workaround will be removed.
